connectors/sources/jira.py

```diff
@@ -292,38 +322,44 @@ def get_default_configuration(cls):
                 "label": "Jira Server username",
                 "order": 2,
                 "type": "str",
+                "value": "admin",
```

Comment:
These values were deleted on purpose by @navarone-feekery. They should not return

Reply:
As way of an explanation: we have removed all arbitrary `value` fields, as it's much cleaner in the UI to not have preset values. All values will now default to a type-safe empty value, so you can safely leave them out of configurable fields now.

Reply:
Oh! seems like some code snippets got reverted. Thanks for pointing this out @artem-shelkovnikov @navarone-feekery. Let me check and get it synced with the original code.